tests/testthat/test-DownloadReportModule.R

```diff
@@ -74,68 +67,51 @@ testthat::test_that("download_report_button_srv - subset of rmd_yaml_args", {
       shiny::testServer(
         download_report_button_srv,
         args = list(reporter = reporter, rmd_yaml_args = rmd_yaml_args_wrong[[iset]]),
-        expr = {
-        }
+        expr = {}
       ),
       "Assertion"
     )
   }
 })
 
-card1 <- ReportCard$new()
-card1$append_text("Header 2 text", "header2")
-card1$append_text("A paragraph of default text", "header2")
-card1$append_plot(
-  ggplot2::ggplot(iris, ggplot2::aes(x = Petal.Length)) +
-    ggplot2::geom_histogram()
-)
-
-reporter <- Reporter$new()
-reporter$append_cards(list(card1))
-
 testthat::test_that("download_report_button_ui - returns a tagList", {
-  testthat::expect_true(
-    inherits(download_report_button_ui("sth"), c("shiny.tag.list", "list"))
-  )
+  checkmate::expect_multi_class(download_report_button_ui("sth"), c("shiny.tag.list", "shiny.tag"))
 })
```

Comment:
I think we should remove this test as it only checks the return value of an `ui` function